.ci/Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG ELASTIC_STACK_VERSION
```

Comment:
maybe we can remove this (later) since I do not see any relevant customizations over the base:
(most of the changes are due updates in the [base file](https://github.com/logstash-plugins/.ci/blob/1.x/Dockerfile) - nothing stands out the base would not do)
```diff
diff --git Dockerfile Dockerfile
index c75ddf0..a1a1708 100644
--- Dockerfile
+++ Dockerfile
@@ -1,19 +1,11 @@
 ARG ELASTIC_STACK_VERSION
+# TODO: refactor this to be implicitly resolved by logstash-plugins/.ci/Dockerfile
 ARG DISTRIBUTION_SUFFIX
-FROM docker.elastic.co/logstash/logstash${DISTRIBUTION_SUFFIX}:${ELASTIC_STACK_VERSION}
-# install and enable password-less sudo for logstash user
-# allows modifying the system inside the container (using the .ci/setup.sh hook)
-USER root
-RUN yum install -y sudo
-RUN usermod -aG wheel logstash && \
-    echo "logstash ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/logstash && \
-    chmod 0440 /etc/sudoers.d/logstash
+FROM docker.elastic.co/logstash/logstash$DISTRIBUTION_SUFFIX:$ELASTIC_STACK_VERSION
 USER logstash
-# whole . plugin code could be copied here but we only do that after bundle install,
-# to speedup incremental builds (locally one's mostly changing lib/ and spec/ files)
-COPY --chown=logstash:logstash Gemfile *.gemspec VERSION* version* /usr/share/plugins/plugin/
+COPY --chown=logstash:logstash Gemfile /usr/share/plugins/plugin/Gemfile
+COPY --chown=logstash:logstash *.gemspec VERSION* version* /usr/share/plugins/plugin/
 RUN cp /usr/share/logstash/logstash-core/versions-gem-copy.yml /usr/share/logstash/versions.yml
-# NOTE: since 8.0 JDK is bundled as part of the LS distribution under $LS_HOME/jdk
 ENV PATH="${PATH}:/usr/share/logstash/vendor/jruby/bin:/usr/share/logstash/jdk/bin"
 ENV LOGSTASH_SOURCE="1"
 ENV ELASTIC_STACK_VERSION=$ELASTIC_STACK_VERSION
@@ -25,8 +17,6 @@ ARG INTEGRATION
 ENV INTEGRATION=$INTEGRATION
 RUN gem install bundler -v '< 2'
 WORKDIR /usr/share/plugins/plugin
-COPY --chown=logstash:logstash .ci/* /usr/share/plugins/plugin/.ci/
-RUN .ci/setup.sh
 RUN bundle install --with test ci
-COPY --chown=logstash:logstash . /usr/share/plugins/plugin/
-RUN bundle exec rake vendor
+COPY --chown=logstash:logstash . /usr/share/plugins/plugin
+RUN bundle exec rake vendor
\ No newline at end of file

```

Reply:
The base Dockerfile runs the `.ci/setup.sh` that this doesn't execute. I think that's matter for another small follow-up PR.